examples/torch/findAndAvoid/controllers/supervisor/supervisor.py

```diff
@@ -0,0 +1,229 @@
+import math
+
+import numpy as np
+
+import utilities as utils
+from deepbots.supervisor.controllers.supervisor_emitter_receiver import \
+    SupervisorCSV
+from deepbots.supervisor.wrappers.keyboard_printer import KeyboardPrinter
+from deepbots.supervisor.wrappers.tensorboard_wrapper import TensorboardLogger
+from models.default_networks import DefaultDDPG
+from models.networks import DDPG
```

Comment:
Are both DefaultDDPG and DDPG needed? As far as i can tell only DDPG is used, default_networks file probably needs to be deleted altogether.